pkg/registry/apis/datasource/standalone.go

```diff
@@ -40,21 +58,62 @@ func NewStandaloneDatasource(group string) (*DataSourceAPIBuilder, error) {
 	if !exists {
 		return nil, fmt.Errorf("plugin %s not found", pluginID)
 	}
-
-	var testsDataQuerierFactory QuerierFactoryFunc = func(ctx context.Context, ri common.ResourceInfo, pj plugins.JSONData) (Querier, error) {
-		return NewDefaultQuerier(ri, td.JSONData, testdatasource.ProvideService(), dsService, dsCache), nil
-	}
-
 	return NewDataSourceAPIBuilder(
 		td.JSONData,
-		NewQuerierProvider(testsDataQuerierFactory),
-		&TestDataPluginContextProvider{},
+		testdatasource.ProvideService(), // the client
+		&testdataPluginConfigProvider{
```

Comment:
^^ this does not really work anymore... but i think it is OK.  the `if true {` statement above is the useful one for testing deployments and execution until we have real multi-tenant access